app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -224,7 +224,7 @@ private void loadSubscriptions() {
                     progressBar.setVisibility(View.GONE);
                 }, error -> Log.e(TAG, Log.getStackTraceString(error)));
 
-        if (UserPreferences.getFeedFilter() != UserPreferences.FEED_FILTER_NONE) {
+        if (UserPreferences.getSubscriptionsFilter().areSubscriptionsFiltered()) {
```

Comment:
The method name sounds a bit strange to me (having the words "subscription filter" twice). I currently do not have a better name in mind, though.

Reply:
How do you feel about using `isEnabled()` instead?